admin/services/data-access/user.data.service.js

```diff
@@ -24,6 +24,36 @@ const userDataService = {
     const rows = await sqlService.query(sql, [paramIdentifier])
     return R.head(rows)
   },
+
+  /**
+   * Find a User information including role and school by the identifier
+   * String -> {User} || undefined
+   * @param identifier
+   * @return {Promise<object>}
+   */
+  sqlFindUserInfoByIdentifier: async (identifier) => {
+    const paramIdentifier = { name: 'identifier', type: TYPES.NVarChar, value: identifier }
+    const sql = `
+      SELECT
+        u.id,
+        u.passwordHash,
+        u.identifier,
+        r.title AS roleName,
+        u.school_id AS schoolId,
+        s.dfeNumber,
+        sce.timezone
+      FROM mtc_admin.${table} u
```

Comment:
We have decided to drop this variable and use the string of the table name instead.

Reply:
@jon-shipley updated to use the table name.